RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -221,26 +267,61 @@
         maxLostLayers = [4,3,2],
         maxDz         = [0.5,0.35,0.2],
         maxDr         = [0.3,0.2,0.1]
-	),
+    ),
     vertices = 'firstStepGoodPrimaryVertices'
 )
-from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
+jetCoreRegionalStepBarrel = jetCoreRegionalStep.clone(
+    src = 'jetCoreRegionalStepBarrelTracks',
+    mva = dict(
+#	minPixelHits  = [1,1,1], # they could be easily increased to at least 2 or 3 !
+        min3DLayers   = [1,2,2],
+    ),
+)
+jetCoreRegionalStepEndcap = jetCoreRegionalStep.clone(
+    src = 'jetCoreRegionalStepEndcapTracks',
+)
 
+from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
 trackingPhase1.toReplaceWith(jetCoreRegionalStep, TrackMVAClassifierPrompt.clone(
      mva = dict(GBRForestLabel = 'MVASelectorJetCoreRegionalStep_Phase1'),
      src = 'jetCoreRegionalStepTracks',
      qualityCuts = [-0.2,0.0,0.4]
 ))
 
+trackingPhase1.toReplaceWith(jetCoreRegionalStepBarrel, TrackMVAClassifierPrompt.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorJetCoreRegionalStep_Phase1'),
+     src = 'jetCoreRegionalStepBarrelTracks',
+     qualityCuts = [-0.2,0.0,0.4]
+))
+trackingPhase1.toReplaceWith(jetCoreRegionalStepEndcap, TrackMVAClassifierPrompt.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorJetCoreRegionalStep_Phase1'),
+     src = 'jetCoreRegionalStepEndcapTracks',
+     qualityCuts = [-0.2,0.0,0.4]
+))
+
 from RecoTracker.FinalTrackSelectors.TrackTfClassifier_cfi import *
 from RecoTracker.FinalTrackSelectors.trackSelectionTf_cfi import *
 trackdnn.toReplaceWith(jetCoreRegionalStep, TrackTfClassifier.clone(
      src = 'jetCoreRegionalStepTracks',
      qualityCuts = qualityCutDictionary["JetCoreRegionalStep"],
 ))
+trackdnn.toReplaceWith(jetCoreRegionalStepBarrel, TrackTfClassifier.clone(
+     src = 'jetCoreRegionalStepBarrelTracks',
+     qualityCuts = qualityCutDictionary["JetCoreRegionalStep"],
+))
+trackdnn.toReplaceWith(jetCoreRegionalStepEndcap, TrackTfClassifier.clone(
+     src = 'jetCoreRegionalStepEndcapTracks',
+     qualityCuts = qualityCutDictionary["JetCoreRegionalStep"],
+))
 
 fastSim.toModify(jetCoreRegionalStep,vertices = 'firstStepPrimaryVerticesBeforeMixing')
 
+
+
+
+
+
+
```

Comment:
```suggestion
```
unclear why more whitespace is needed here

Reply:
not sure what you mean by "still pending"
seed cannot be merged, because the building is done differently